sdk/go/common/workspace/project.json

```diff
@@ -324,7 +312,8 @@
                 "secret":{
                     "type":"boolean"
                 },
-                "default":{ }
+                "default":{ },
+                "value": { }
```

Comment:
I think we should split `configTypeDeclaration` into `configTypeDeclaration` without `value` and `configValueDeclaration` with _just_ `value`

Reply:
I tried this and refactored the code but the JSON validation errors are a lot more verbose than the errors returned by the current logic. So I reverted it back to this.